docs/source/docker/overview.rst

```diff
@@ -61,7 +63,7 @@ We worked hard to make this happen and are very proud to present you with the fo
   
   * 8 CPU cores or more
 
-  * 16GB free RAM
+  * 16GB free RAM (>32GB for datasets above 10m samples)
```

Comment:
It's even more, right? I required 40GB for 5 million samples.

I believe millions are indicated with a capital "M". Alternatively you can just write "million".

Reply:
Changed to 64GB